### DISCUSSION TITLE
can I make write only user API token for [Sep 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3749] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### You can create a shared form view and use the shared form view api to submit the data  
#  
> 👨‍💻 **pranavxc**  
#### You can't create an api token specific to a table/view and which is write only  
#  
> 👨‍💻 **pranavxc**  
#### a workaround will be the shared form view  
#  
> 🙋‍♂️ **emdax**  
#### thanks, where are docs to shared form view ?  
#  
> 🙋‍♂️ **emdax**  
#### how can i POST to the form view in JS ?  
#  
> 🙋‍♂️ **emdax**  
#### doing POST gives CORS error  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016865619460763749)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
